### PR TITLE
Added vaex recipe

### PR DESCRIPTION
**Pull request by maartenbreddels**

### CODE REVIEW

**Comment by jakirkham** [comment]
Please clean out the example comments.

File: recipes/vaex/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    - aplus
+    - tornado >4.1
+    - futures
+    - future >=0.15.2
+    - pyyaml
+    - cachetools
+    - progressbar2
+    - attrdict
+    - pandas
+    - pillow
```
Comment (jakirkham) [inline recipes/vaex/meta.yaml]: Please drop all of these that are not strictly needed at build time. Likely just `python` and `setuptools`.

File: recipes/vaex/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,103 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "vaex" %}
+{% set version = "1.0.0b2" %}
```
Comment (jakirkham) [inline recipes/vaex/meta.yaml]: Is there a non-beta release that we can use?

File: recipes/vaex/meta.yaml (hunk 1 above)
Comment (maartenbreddels) [inline recipes/vaex/meta.yaml]: 👍

File: recipes/vaex/meta.yaml (hunk 2 above)
Comment (maartenbreddels) [inline recipes/vaex/meta.yaml]: There is an old one, but it's better to wait till the 1.0.0 release. I wanted to see how this all works, is that ok? 
Btw, how to treat dependencies that I'm not the author of, shall I write the recipes myself and make a PR on a different branch?

**Comment by 183amir** [comment]
@maartenbreddels could you clean this up too?

**Comment by jakirkham** [comment]
I'm trying to make an infrastructure fix. So I need hold the AppVeyor build of this until we are done. Will restart when it is complete.

**Comment by jakirkham** [comment]
Trying to make an infrastructure fix. So need to hold the AppVeyor build of this until completed. Will restart after.

**Comment by maartenbreddels** [comment]
Ok, you mean it does not skip the win build automagically? So does that mean this can be merged?

**Comment by jakirkham** [comment]
If you don't need Windows, could you please add `[skip appveyor]` in your commit messages?

**Comment by maartenbreddels** [comment]
took a few attempts (some gateway issues..), but it builds!

File: recipes/vaex/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    - vaex.file
+    - vaex.misc
+    - vaex.test
+    - vaex.ui
+    - vaex.ui.icons
+    - vaex.ui.plugin
+
+about:
+  home: http://vaex.astro.rug.nl
+  license: MIT
```
Comment (jakirkham) [inline recipes/vaex/meta.yaml]: Is there a [license file]( http://conda.pydata.org/docs/building/meta-yaml.html#license-file ) we can include?

Comment (jakirkham) [inline recipes/vaex/meta.yaml]: Also it would be nice to add `license_family: MIT`.

Comment (maartenbreddels) [inline recipes/vaex/meta.yaml]: The LICENSE file wasn't in the manifest file, I've added it and will include it in the recipe next release.

**Comment by maartenbreddels** [comment]
hope it is ready to merge now

**Comment by 183amir** [comment]
@maartenbreddels Your last commit seems to be empty.

**Comment by maartenbreddels** [comment]
Sorry for that @183amir, fixed now :+1:

**Pull request merged**